modules/programs/zsh.nix

```diff
@@ -261,7 +281,7 @@ in
       programs.zsh.enableCompletion = mkDefault true;
 
       home.file = map (plugin: {
-        target = ".zsh/plugins/${plugin.name}";
+        target = relToDotDir "plugins/${plugin.name}";
```

Comment:
This will put zsh plugins into a `plugins` folder inside $HOME directory by default ( `~/plugins`), which doesn't look that descriptive anymore.

Reply:
Oh right